nebula/ui/components/forms/MZTextField.qml

```diff
@@ -56,8 +56,8 @@ TextField {
     onTextChanged: {
         if (Qt.platform.os === "osx") {
             textField.focusReasonA11y = true
-            textField.focus = false;
-            textField.forceActiveFocus();
+            Accessible.focused = false
```

Comment:
Might be worthwhile to follow this up with a helpful code comment to our future selves explaining why we set `Accessible.focused` to false on L59, then to true on the next.

Reply:
There is a comment by Florian right above `onTextChanged` that explains this weird workaround :D